src/db/migrations/20240822032133_subscriber_email_preferences.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export async function up(knex) {
+  await knex.schema
+    .createTable('subscriber_email_preferences', function(table) {
+      table.increments('id').primary();
+      table.integer("subscriber_id").references("subscribers.id").unique().notNullable().onDelete('CASCADE').onUpdate('CASCADE');
+      table.boolean('instant_breach_alert').defaultTo(true);
+      table.boolean('all_emails_to_primary').defaultTo(true);
+      table.boolean('monthly_monitor_report').defaultTo(true);
```

Comment:
I recall reading (but can't find) the advice to instead make this a nullable datetime column `*_disabled_at`, would that make sense here? It's not a table that's read or written to particularly often I don't think, so not too much of a load, but would e.g. allow us to notice weird unsubscription patterns, and/or correlate unsubscription spikes with certain events? (Alternatively, simple Glean metrics might be fine too.)